src/tests/functional/inference_engine/transformations/smart_reshape/lstm_states_broadcast.cpp

```diff
@@ -0,0 +1,106 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <openvino/core/model.hpp>
+#include <openvino/opsets/opset9.hpp>
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+
+template<class T>
+std::shared_ptr<ov::opset9::Constant> create_constant(const std::vector<T>& data, const ov::element::Type_t et = ov::element::i64, bool scalar = false) {
+    ov::Shape shape = scalar ? ov::Shape{} : ov::Shape{data.size()};
+    return ov::opset9::Constant::create(et, shape, data);
+}
+
+std::shared_ptr<ov::opset9::Constant> create_zero_constant(const ov::element::Type_t et, ov::PartialShape shape) {
```

Comment:
This is similar this tests and also in reshape_sinking tests, can move as some test util as  overload set?